pos_user_restriction/models/pos_config.py

```diff
@@ -1,12 +1,25 @@
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class PosConfig(models.Model):
     _inherit = "pos.config"
+
     assigned_user_ids = fields.Many2many(
         "res.users",
         string="Assigned users",
         help="Restrict some users to only access their assigned points of sale. "
         "In order to apply the restriction, the user needs the "
         "'User: Assigned POS Only' group",
     )
+    set_pos_user_group = fields.Boolean(
```

Comment:
Why we need new field?

Reply:
switch the cashier according to the code of the Owl Structure in  v14  for the switch cashier (employee) calls the (pos_user_group) method to switch cashier easily  and  can start a session so I have put a new field and method

Reply:
This is not ok.

Try making the `group_pos_user_id` field a computed field instead, it should probably work

Reply:
@ivantodorovich Thanks for the review suggestions are implemented you can re-review it.